src/App.js

```diff
@@ -66,13 +66,18 @@ import { walletConnectLoadState } from './redux/walletconnect';
 import { rainbowTokenList } from './references';
 import { branchListener } from './utils/branch';
 import { analyticsUserIdentifier } from './utils/keychainConstants';
+import {
+  CODE_PUSH_DEPLOYMENT_KEY,
+  isCustomBuild,
+} from '@rainbow-me/handlers/fedora';
 import { SharedValuesProvider } from '@rainbow-me/helpers/SharedValuesContext';
 import Routes from '@rainbow-me/routes';
 import logger from 'logger';
 import { Portal } from 'react-native-cool-modals/Portal';
-
 const WALLETCONNECT_SYNC_DELAY = 500;
 
+let FedoraToastRef;
```

Comment:
why not React.createRef? This will work better for the ref prop than a function with assigning.